### DISCUSSION TITLE
Highlight post activity in widget

### DISCUSSION THREAD

**Discussion by ryancrumble** [dailydotdev/daily#565] [Feature Requests]
An option in the settings to enable/disable badges on the icons in the widget. 
I take benefit from seeing the activity on a post in the daily.dev and I think it can be useful in the widget.
The design may appear a little busy, so it might be something to play around with in the design.

This could also be extended to add a 'hot' tab, like what is currently present on cards in the app.

Also, to consider having it as an optional feature that the user can toggle.

**Comment by idoshamun** [comment]
Hi @ryancrumble,

Thanks for the suggestion. Just to make sure I understand, do you suggest adding the actual numbers of upvotes and comments in the companion widget?

**Reply by ryancrumble** [reply]
Hi @idoshamun 

That is correct. Adding the like count, message count, or even the 'hot' tag.

**Reply by idoshamun** [reply]
Thanks, we will probably consider it for v2 :)
